src/components/tx/SignOrExecuteForm/index.tsx

```diff
@@ -98,6 +99,8 @@ const SignOrExecuteForm = ({
   const onSign = async (): Promise<string> => {
     const [connectedWallet, createdTx] = assertSubmittable()
 
+    trackEvent(WALLET_EVENTS.OFF_CHAIN_SIGNATURE)
```

Comment:
I would suggest to track this and the other event in this file from tx events instead. This component is already overloaded with logic.